### DISCUSSION TITLE
Play 2.8 with Akka 2.7 or higher

### DISCUSSION THREAD

**Discussion by aniketskulkarni** [playframework/playframework#11755] [General]
Hello,
I have a few questions regarding the usages of Play 2.8 with Akka 2.7 or higher.
- What is the roadmap for using Play framework? The latest release in Jan 23 states that “…news coming soon…“.
- Do you intend to stay in Akka 2.6 even past its end of life?
- Do you expect an Akka fork with a friendlier license to appear?
- Do you plan to move away from Akka?
Thanks!

**Comment by mkurz** [comment]
> What is the roadmap for using Play framework? The latest release in Jan 23 states that “…news coming soon…“.

The target is to release Play 2.9 beginning of May, after Scala 3.3.0, 2.13.11 and 2.12.18 are released.
Major work of 2.9 is done, most important migration docs need to be updated. There are still other issues and PR that need to I want to take a look at, but IMHO nothing show stopping.

> Do you intend to stay in Akka 2.6 even past its end of life?

After Play 2.9 will will work to release another major version around September / October (when akka 2.6.x is EOL) to solve the akka(-http) license problem. Until that we will stay on akka 2.6.x and akka-http 10.2.x.

> Do you expect an Akka fork with a friendlier license to appear?

That exists already and is called pekko:
* https://pekko.apache.org/
* https://github.com/apache/incubator-pekko
* https://github.com/apache/incubator-pekko-http

> Do you plan to move away from Akka?

No final decision has been made. After 2.9 this will be evaluated in deepth.

**Comment by mkurz** [comment]
@aniketskulkarni Meanwhile Play 2.9 and 3.0 have been released now: https://twitter.com/playframework/status/1721576472047911007
You can use Play 2.9 and upgrade to latest akka 2.9.0 and akka-http 10.6.0 (as of today). Or you can use Play 3.0 and switch to Pekko.
Please find more about all that here:
- https://www.playframework.com/documentation/3.0.x/General#How-Play-Deals-with-Akkas-License-Change
- Play 2.9: https://github.com/playframework/playframework/releases/tag/2.9.0
- Play 3.0: https://github.com/playframework/playframework/releases/tag/3.0.0


- Play 2.9 highlights: https://www.playframework.com/documentation/latest/Highlights29
- Play 2.9 migration guide: https://www.playframework.com/documentation/latest/Migration29
- Play 3.0 highlights: https://www.playframework.com/documentation/latest/Highlights30
- Play 3.0  and migration guide: https://www.playframework.com/documentation/latest/Migration30

Update akka in Play 2.9:
- Play Scala app: https://www.playframework.com/documentation/2.9.x/ScalaAkka#Updating-Akka-version
- Play Java app: https://www.playframework.com/documentation/2.9.x/JavaAkka#Updating-Akka-version
